app/components/Modals/SendModal/ReadCode/ReadCode.scss

```diff
@@ -0,0 +1,65 @@
+@import '../../../../styles/variables';
+
+.qrCodeScannerSection{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  
+    video{
+        width: 230px;
+    }
+
+    button {
+        margin-top: 16px;
+    }
+}
+
+.placeholderFrameLine{
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    border-style: solid;
+}
+
+.qrCodeScannerPlaceholder{
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 170px;
+    border-radius: 5px;
+    background-color: $light-grey;
+
+    :nth-child(1){
```

Comment:
doing it this way is great but its a bit less explicit to know that the divs are being styled here. In general, I think everything should be as clear as possible in an open source project; however I think its ok to do this if you don't want to change it

Reply:
I went back and forth on this a few times when I was writing it. I agree that explicit is probably better in this context. Updated.